osu.Game.Rulesets.Soyokaze/Mods/SoyokazeModHidden.cs

```diff
@@ -13,6 +15,19 @@ namespace osu.Game.Rulesets.Soyokaze.Mods
 {
     public class SoyokazeModHidden : ModHidden
     {
+        [SettingSource("Fading Approach Circle", "Allow approach circles to fade instead of disappearing")]
+        public Bindable<bool> FadingApproachCircle { get; } = new BindableBool(false);
+
+        [SettingSource("Fading Approach Circle Speed", "Adjust the fading speed of approach circles")]
+        public Bindable<double> FadingApproachCircleSpeed { get; } = new BindableDouble
+        {
+            Precision = 0.05f,
+            MinValue = 0.5,
+            MaxValue = 2.0,
+            Default = 1.0,
+            Value = 1.0
+        };
```

Comment:
I don't like this being an option. While it might be convenient to be able to change the approach circle fade speed, I think it makes more sense to have this fade speed tied to `fadeOutDuration`. It's just simpler and more consistent that way for the player. Having more options isn't always better.